2-mandatory/3-groceries.js

```diff
@@ -27,13 +27,22 @@ Exercise 1:
 */
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
+ for(let values of Object.values(weeklyMealPlan)){
+   
+   weeklyGroceriesToBuy.push(values)
```

Comment:
Unfortunately, this code is just pushing all meal plans into the array. The requirement is that the items shouldn't be repeating